Documentation/BuildInstructionsMacOS.md

```diff
@@ -13,7 +13,7 @@ Make sure you also have all the following dependencies installed:
 
 ```console
 # core
-brew install coreutils e2fsprogs qemu bash imagemagick ninja cmake ccache rsync zstd
+brew install coreutils e2fsprogs qemu bash imagemagick ninja cmake ccache rsync zstd gsed
```

Comment:
I thought we were compatible with macOS's native `sed`, why do we need `gsed`? Please explain in the commit message.

Reply:
When building with the clang toolchain it said that it didn't find the gsed executable. I'm guessing using just sed instead of gsed should work, but haven't tested it. Don't know if we're using some special gsed only flags.

Reply:
It looks like that specific dependency was added in PR #11578. I've just tried building clang by replacing `gsed` with `sed` and it looks like everything worked just fine. Can you confirm?

Reply:
It build for me as well without `gnu-sed`, so [removing](https://github.com/SerenityOS/serenity/pull/21031) the dependency seems more sensible.

Reply:
Seems to work just fine with `SED="sed"`.